src/lib.rs

```diff
@@ -478,7 +483,10 @@ pub struct UnixTime(u64);
 
 impl UnixTime {
     /// The current time, as a `UnixTime`
-    #[cfg(feature = "std")]
+    #[cfg(any(
+        feature = "std",
+        all(target_family = "wasm", target_os = "unknown", feature = "web")
+    ))]
```

Comment:
> > Obviously this would mean users who target `wasm32-unknown-unknown` without the feature will get a panic when they hit this part of the code.
> 
> Could/should we make this a compile time error?

Maybe guarding it would be an option as well:
```suggestion
    #[cfg(any(
        all(feature = "std", not(all(target_family = "wasm", target_os = "unknown"))),
        all(target_family = "wasm", target_os = "unknown", feature = "web")
    ))]
```

Reply:
Are you sure? Isn't it possible to run target wasm32_unknown_unknown in a non-web environment with std feature where std is available?

Reply:
Calling `SystemTime::now()` will always panic when running it in `wasm32-unknown-unknow`. Unfortunately there is no way around this currently.

See the [Rust Std source code on this](https://github.com/rust-lang/rust/blob/07dca489ac2d933c78d3c5158e3f43beefeb02ce/library/std/src/sys/wasm/mod.rs#L47-L48).